lib/esformatter.js

```diff
@@ -42,6 +43,13 @@ exports.unregisterAll = plugins.unregisterAll;
 
 
 function format(str, opts) {
+  if (opts && opts.pipe) {
+    str = pipe.run(opts.pipe, str).toString();
```

Comment:
In fact we need to support `before` and `after` to make it more flexible (like we do with the plugins)